dist/css/style.css

```diff
@@ -123,3 +123,507 @@ h1 {
   padding-left: 0;
 }
 
+/* Works and Projects */
+.works-card {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 8rem;
+  padding: 12rem 0;
+  background: #e5e5e5;
+}
+  
+.grid-container,
+.card-container,
+.text-container {
+  display: flex;
+  flex-direction: column;
+}
+  
+.grid-container,
+.card-container,
+.text-container,
+.card-title,
+.frame-two {
+  background-color: #ffff;
+}
+  
+.grid-container {
+  margin: 0 2.4rem;
+  max-width: 100%;
+  gap: 2.4rem;
+  border: 0.1rem solid #dfe1e6;
+  border-radius: 1.6rem;
+  width: 34.9rem;
+}
+  
+.card-container {
+  align-items: center;
+  justify-content: center;
+  margin: 5%;
+  padding: 0 1rem;
+}
+  
+.text-container {
+  justify-content: center;
+  align-items: flex-start;
+  gap: 1.2rem;
+}
+  
+.card-title {
+  font-family: 'Poppins', sans-serif;
+  font-size: 3.2rem;
+  line-height: 4.4rem;
+  font-weight: 700;
+  color: #172b4d;
+}
+  
+.frame-two {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 1.2rem;
+  width: 23.6rem;
+  height: 3.2rem;
+}
+
+.client,
+.card-description,
+.role,
+.small-btn {
+  color: #344563;
+  background-color: #ffff;
+}
+
+.client {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0.8rem 0;
+  width: 5.5rem;
+  height: 3.2rem;
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [ ] Great job so far. Before you can use the font-family used in the design which is  `Poppins`, you need to add it to he head section of your `index.html` or import it inside your `style.css` file. Kindly import the font from [googlefont](https://fonts.google.com/specimen/Poppins?query=poppins). currently it does not exist in your file, so the one used is the fallback option which is sans-serif. Kindly apply it to every other areas where applicable.